webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -985,6 +1271,28 @@ private TextUnitSearcherParameters baseParams(
     return result;
   }
 
+  private TextUnitSearcherParameters baseParams(
```

Comment:
Hmm - baseParams implementations look like they would benefit from the Builder pattern here - or, actually, even better, if TextUnitSearcherParameters would implement that - we wouldn't need all the differenet baseParams methods here. (not strictly needed in this PR though)

Reply:
Thank you, @maallen - I'll close this thread for now.